utils.cpp:82:63: error: use of undeclared identifier 'errno' on macOS

On macOS the build of V0.2.3 fails with the following error:
```
==> make -C src
mkdir ../obj
g++ -Wall -std=c++11 -g -c -fPIC -o ../obj/utils.o utils.cpp
utils.cpp:82:63: error: use of undeclared identifier 'errno'
                throw stringf("Failed to read line from file: %s", strerror(errno));
                                                                            ^
1 error generated.
make: *** [../obj/utils.o] Error 1
```

This is a successful fix:
```
    inreplace "src/utils.cpp", "#include <unistd.h>",
                               "#include <unistd.h>\n#include <errno.h>"
```

--- a/src/utils.cpp
+++ b/src/utils.cpp
@@ -11,6 +11,7 @@
 #include <cstdlib>
 #include <climits>
 #include <unistd.h>
+#include <errno.h>
 
 
 string vstringf(const char* format, va_list va)

